.azure-pipelines-steps.yml

```diff
@@ -0,0 +1,27 @@
+steps:
+- script: npm i -g npm@$(npm_version)
+  displayName: Use legacy npm version $(npm_version)
+  condition: ne(variables['npm_version'], '')
+
+- task: NodeTool@0
+  inputs:
+    versionSpec: '$(node_version)'
+  displayName: Use Node $(node_version)
```

Comment:
What does this do?  Just print the node version?

Reply:
It installs the specified version of node. Its similar to saying `nodejs: {node_version}` in Travis, we're just drawing the node version from the variable we defined in `.azure-pipelines.yml`

Reply:
Ah, okay.  So the `NodeTool@0` task is installing it.  That makes sense.